modular_zubbers/code/modules/status_indicators/status_indicator.dm

```diff
@@ -81,10 +77,6 @@ GLOBAL_LIST_INIT(potential_indicators, list(
 	UnregisterSignal(attached_mob, COMSIG_LIVING_STATUS_IMMOBILIZE)
 	UnregisterSignal(attached_mob, COMSIG_LIVING_STATUS_UNCONSCIOUS)
```

Comment:
```suggestion
	UnregisterSignals(attached_mob, list(COMSIG_LIVING_LIFE, COMSIG_LIVING_STATUS_STUN, COMSIG_LIVING_STATUS_KNOCKDOWN, COMSIG_LIVING_STATUS_PARALYZE, COMSIG_LIVING_STATUS_IMMOBILIZE, COMSIG_LIVING_STATUS_UNCONSCIOUS))
```
(Remove all the unregistersignal lines when you do this github just wont let me multi line review it)